src/components/Breadcrumbs.tsx

```diff
@@ -32,10 +29,25 @@ const Breadcrumbs = ({ className, crumbs, dynamic }: BreadcrumbsProps) => {
     return items;
   };
 
+  const getPrevPath = () => {
+    const name = routing.find(item => {
+      if(item.path === prevPath || item.path.includes(prevPath)) {
```

Comment:
Seems to be fine with just `item.path.includes(prevPath)`

The function might looks like 
`const { name  } = routing.find(({ path }) => path.includes(prevPath)) || { name: '' };

setPrevPathName(name);
`